WiserTaskScheduler/WiserTaskScheduler/Core/Services/SlackChatService.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Threading.Tasks;
+using GeeksCoreLibrary.Core.DependencyInjection.Interfaces;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using WiserTaskScheduler.Core.Interfaces;
+using WiserTaskScheduler.Core.Models;
+using WiserTaskScheduler.Modules.Slack.modules;
+using SlackNet;
+using SlackNet.WebApi;
+
+namespace WiserTaskScheduler.Core.Services
+{
+    public class SlackChatService : ISlackChatService, ISingletonService
+    {
+        private const string LogName = "SlackChatService";
+        
+        private readonly IServiceProvider serviceProvider;
+        private readonly SlackSettings slackSettings;
+
+        public SlackChatService(IServiceProvider serviceProvider, IOptions<WtsSettings> logSettings)
+        {
+            this.serviceProvider = serviceProvider;
+            this.slackSettings = logSettings.Value.SlackSettings;
+        }
+
+#if !DEBUG
```

Comment:
Currently the Slack messages will only be send when in debug modus instead of production. I think you forgot to revert this after testing?

Reply:
🤦‍♂️ yeah.. i did, thanks